purldb-toolkit/src/purldb_toolkit/purlcli.py

```diff
@@ -346,21 +333,17 @@ def get_urls_details(purls, output, file, head, command_name):
 
         urls_purl = check_urls_purl(purl)
```

Comment:
I think `urls_purl` should be renamed `purl_status` or something that would indicate what the return value is

Reply:
Another good clarification -- done.